composer.json

```diff
@@ -185,6 +187,12 @@
             },
             "drupal/publication_date": {
                 "Issue #2983348: Published on empty for existing nodes.": "patches/publication_date-fill_published_at-2983348-60-for-composer.patch"
+            },
+            "drupal/pathologic": {
+                "Issue #758118: wild card for Additional paths.": "patches/pathologic-758118-wildcard-for-additional-paths-18.patch"
```

Comment:
This patch is needed so that we don't have to specify each full prison hostname.  Instead we can use `https://*.content-hub.prisoner.service.justice.gov.uk`

The pathologic module needs to know which hostnames to convert (from absolute to relative).  It won't just convert everything.